docs/src/adoc/index.adoc

```diff
@@ -352,12 +385,13 @@ Document document =
 handle.createUpdate("insert into documents (id, folder_id, name, contents) " +
                     "values (:d.id, :f.id, :d.name, :d.contents)")
       .bindBean("f", folder)
-      .bindBean("d", document)
+      .bindMethods("f", folder)
+      .bindFields("d", document)
       .execute();
 ----
 
 [WARNING]
-Neither `bindMap()` nor `bindBean()` support binding of nested properties (e.g.
+Neither `bindMap()`, `bindMethods()`, `bindFields()` nor `bindBean()` support binding of nested properties (e.g.
```

Comment:
Needs an Oxford comma between ``` `bindFields()` ``` and `nor`.

Also.. I know this is super pedantic, but can you line wrap this to follow 80-character line limit? I've been working carefully to keep the documentation format consistent.

Please read through the "Style guidelines" comment block at the top of this file if you haven't already, so you're aware of them.